src/domain/event/Event.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { FunctionComponent } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Formik } from 'formik';
+import { useHistory } from 'react-router';
+
+import Icon from '../../common/components/icon/Icon';
+import styles from './event.module.scss';
+import PageWrapper from '../app/layout/PageWrapper';
+import personIcon from '../../assets/icons/svg/person.svg';
+import backIcon from '../../assets/icons/svg/arrowLeft.svg';
+import EnhancedInputField from '../../common/components/form/fields/input/EnhancedInputField';
+import SelectField from '../../common/components/form/fields/select/SelectField';
+import Button from '../../common/components/button/Button';
+
+interface SignupValues {
+  date: string;
+  time: string;
+}
+
+const Event: FunctionComponent = () => {
+  const history = useHistory();
+  const { t } = useTranslation();
+
+  const initialValues: SignupValues = {
+    date: '',
+    time: '',
+  };
+
+  const handleSubmit = (values: SignupValues) => {
+    // TODO: Do something like querying api again new filters
+    // OR filter in js.
+  };
+
+  // TODO: Get from API (all todo's are pending that)
+  const selectOptions = [
+    { value: 1, label: 'one' },
+    { value: 2, label: 'two' },
+  ];
+  // TODO:
+  const bgImageUrl =
+    'https://edit.myhelsinki.fi/sites/default/files/styles/hero_image/public/2019-12/flow_festival_2019_friday_c_petri_anttila_mg_0397.jpg';
```

Comment:
lol

Reply:
Backend now returns the fully qualified url